scripts/common/install-latest-librdkafka.sh

```diff
@@ -0,0 +1,32 @@
+#!/bin/bash
+#### Install latest librdkafka, right from the GIT repo.
+
+### Define the version of kafka we are pinning to.
+kafka_version=1.5.0
+
+
+### Install librdkafka
+## Install dependencies
+apk add --no-cache libressl-dev || exit 2
```

Comment:
I think these should be installed as virtual and then removed afterwards. Otherwise they will pollute our base image with unnecessary build time dependencies see the apk `--virtual` flag (and I think there are examples in here)

Reply:
I didn't know it could do that! Cool. Let me try that out.